SpatialGDK/Documentation/content/get-started/gdk-template.md

```diff
@@ -0,0 +1,152 @@
+    
+    ![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/schema-button.png)<br/>
+    _Image: On the GDK toolbar in the Unreal Editor select the **Schema** button_<br/>
+
+1. Select the [**Snapshot**]({{urlRoot}}/content/generating-a-snapshot) toolbar button to generate a snapshot.
+
+    ![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/snapshot-button.png)<br/>
+    
+    _Image: On the GDK toolbar in the Unreal Editor select the **Snapshot** button_
+
+
+### Deploy your project 
+
+#### Deploy locally
+
+1. In the Unreal Editor, on the SpatialOS GDK toolbar, select **Start**. This opens a terminal window and starts a local SpatialOS deployment. Wait until you see the output `SpatialOS ready. Access the inspector at http://localhost:21000/inspector` in your terminal window.<br/>
+![Toolbar]({{assetRoot}}assets/set-up-template/template-start.png)<br/>
+_Image: On the GDK toolbar in the Unreal Editor select the **Start** button_<br/>
+1. On the Unreal Editor toolbar, open the **Play** drop-down menu.
+1. Under **Multiplayer Options**, set the number of players to **2** and check the box next to **Run Dedicated Server**. Then, under Modes, select **New Editor Window (PIE)**.<br/>
+![]({{assetRoot}}assets/set-up-template/template-multiplayer-options.png)<br/>
+_Image: The Unreal Engine **Play** drop-down menu, with Multiplayer Options highlighted._    <br/>
+1. On the Unreal Engine toolbar, select **Play** to run the game.<br/>
+1. When you're done, select the **Stop** button in the SpatialOS section of the toolbar to stop your local SpatialOS deployment.<br/>
+![Toolbar]({{assetRoot}}assets/set-up-template/template-stop.png)<br/>
+_Image: On the GDK toolbar in the Unreal Editor select the **Stop** button_<br/>
+
+If you want to run multiple server-workers in the editor, see the [Toolbar documentation]({{urlRoot}}/content/toolbars#launching-multiple-pie-server-workers) regarding launching multiple PIE server-workers
+
+#### Deploy in the cloud
+
+To run a cloud deployment, you need to prepare your server-worker and client-worker [assemblies](https://docs.improbable.io/reference/latest/shared/glossary), and upload them to the cloud.        
+
+> Building the assemblies can take a while - we recommend installing <a href="https://www.incredibuild.com/" data-track-link="Incredibuild|product=Docs|platform=Win|label=Win" target="_blank">IncrediBuild</a> to speed up build times.
+
+##### Step 1: Set up your SpatialOS project name. 
+When you signed up for SpatialOS, your account was automatically associated with an organisation and a project, both of which have the same generated name.
+
+1. Find this name by going to the [Console (SpatialOS Documentation)](https://console.improbable.io). 
+The name should look something like beta_randomword_anotherword_randomnumber. In the example below, it’s beta_yankee_hawaii_621. <br/>
```

Comment:
```suggestion
The name should look something like `beta_randomword_anotherword_randomnumber`. In the example below, it’s `beta_yankee_hawaii_621`. <br/>
```

Reply:
Suggestion accepted and applied. Thanks.

Reply:
Worth noting that this line was lifted from another page. I'm pretty sure its replicated across the Unreal and Unity docs so I'll need to go update the styling to match. 
Later... I'll do that later.